layouts/community/numpad_5x4/layout.json

```diff
@@ -0,0 +1,5 @@
+[{a:7},"","","",""],
```

Comment:
You don't actually need this file here at all.

Reply:
Do you mean, i'm safe to delete  `layouts/community/numpad_5x4/layout.json` file @noroadsleft?